cf-serverd/cf-serverd-functions.c

```diff
@@ -684,6 +688,26 @@ static void AcceptAndHandle(EvalContext *ctx, int sd)
     ServerEntryPoint(ctx, MapAddress(ipaddr), info);
 }
 
+static size_t GetListenQueueSize(void)
+{
+    const char *const queue_size_var = getenv("CF_SERVERD_LISTEN_QUEUE_SIZE");
+    if (queue_size_var != NULL)
+    {
+        long queue_size;
+        int ret = StringToLong(queue_size_var, &queue_size);
+        if ((ret == 0) && (queue_size > 0) && (queue_size <= MAX_LISTEN_QUEUE_SIZE))
+        {
+            return (size_t) queue_size;
+        }
+        Log(LOG_LEVEL_VERBOSE,
+            "$CF_SERVERD_LISTEN_QUEUE_SIZE = '%s' doesn't specify a valid number for listen queue size, "
```

Comment:
maybe INFO level? How often would this be called? Just every time cf-serverd is started from the command line or ?

Reply:
Yeah, that's a good point. This only happens once. It should actually be a warning.